### ISSUE TITLE
NPM package import error.

### ISSUE THREAD

**Issue by edgar-505** [gaoryrt/number-flip#26]
Hi , I tried to import the package `import { Flip } from "number-flip"` to my framer project and this happens :

<img width="692" alt="Screenshot 2023-11-11 at 21 50 40" src="https://github.com/gaoryrt/number-flip/assets/68522816/8870bdad-e444-460e-9363-7f05b6991fc8">

my current hunch is that may be the package is not ESM compatible.
so, is the package ESM compatible?

**Comment by gaoryrt** [comment]
Yes, it is. Maybe you missed to [install](https://github.com/gaoryrt/number-flip#install) the package first?

**Comment by edgar-505** [comment]
> Yes, it is. Maybe you missed to [install](https://github.com/gaoryrt/number-flip#install) the package first?

framer (the app) tries to install the package , but it failed to import the package.
but I found a work-around , so I copied the whole source code from `number-flip.ts`
then paste it to my own file , then I refer my import to my own local file.

the package is not verified in their app , maybe.

**Comment by gaoryrt** [comment]
> > Yes, it is. Maybe you missed to [install](https://github.com/gaoryrt/number-flip#install) the package first?
> 
> framer (the app) tries to install the package , but it failed to import the package. but I found a work-around , so I copied the whole source code from `number-flip.ts` then paste it to my own file , then I refer my import to my own local file.
> 
> the package is not verified in their app , maybe.

I think that [dist/index.js](https://github.com/gaoryrt/number-flip/blob/master/dist/index.js) is suitable for Webpack/Gulp.

**Comment by gaoryrt** [comment]
> so , should I copy the whole thing [dist/index.js](https://github.com/gaoryrt/number-flip/blob/master/dist/index.js) or from [number-filp.ts](https://github.com/gaoryrt/number-flip/blob/master/number-flip.ts) ?

It depends on whether your development environment includes a ts explainer. If it support ts, number-flip.ts is good. But dist/index.js just works for both.
